Demo/Demo/SwiftUIComponents/CardVaultViews/UpdateSetupTokenView.swift

```diff
@@ -0,0 +1,82 @@
+import SwiftUI
+
+struct UpdateSetupTokenView: View {
+
+    let setupToken: String
+
+    @State private var cardNumberText: String = "4111 1111 1111 1111"
+    @State private var expirationDateText: String = "01 / 25"
+    @State private var cvvText: String = "123"
+
+    @ObservedObject var cardVaultViewModel: CardVaultViewModel
+    @ObservedObject var baseViewModel: BaseViewModel
+
+    @State private var isLoading = false
+
+    public init(baseViewModel: BaseViewModel, cardVaultViewModel: CardVaultViewModel, setupToken: String) {
+        self.cardVaultViewModel = cardVaultViewModel
+        self.baseViewModel = baseViewModel
+        self.setupToken = setupToken
+    }
+
+    var body: some View {
+        VStack(spacing: 16) {
+            HStack {
+                Text("Vault Card")
+                    .font(.system(size: 20))
+                Spacer()
+            }
+            CardFormView(
+                cardNumberText: $cardNumberText,
+                expirationDateText: $expirationDateText,
+                cvvText: $cvvText
+            )
+
+            let card = baseViewModel.createCard(
+                cardNumber: cardNumberText,
+                expirationDate: expirationDateText,
+                cvv: cvvText
+            )
```

Comment:
nit: This feels like it may belong in the constructor?

Reply:
I thought that since I am also using baseVM's `getCoreConfig` in this view, I could just have it here so it doesn't clutter up the main `CardVaultView`. I feel like with `CardFormView` here, card being created where the card information is entered makes sense.